packages/links/LinkButton.jsx

```diff
@@ -32,7 +35,10 @@ export function LinkButton({
   return internalUrl || externalUrl ? (
     // eslint-disable-next-line react/jsx-no-target-blank
     <a
-      className="su-group su-inline-block su-button hocus:su-underline md:su-px-30 md:su-pt-12 md:su-pb-14 su-text-18 md:su-text-20 su-transition-colors"
+      className={cnb(
+        "su-group su-inline-block su-button hocus:su-underline md:su-px-30 md:su-pt-12 md:su-pb-14 su-text-18 md:su-text-20 su-transition-colors dark:hocus:su-ring-1 dark:hocus:su-ring-white",
```

Comment:
@broleomo @iamrentman I used the TW ring utility here. It achieves the same effect as border, but it's inset, so doesn't cause a layout shift when going to no border (regular state) to 1px border on hocus in dark mode. It's a handy utility 😄 
https://tailwindcss.com/docs/ring-width#inset-rings